apps/e2e/lookup-storybook-e2e/src/e2e/lookup.component.cy.ts

```diff
@@ -0,0 +1,217 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('lookup-storybook', () => {
+  E2eVariations.forEachTheme((theme) => {
+    describe(`in ${theme} theme`, () => {
+      ['multiple-mode', 'single-mode', 'disabled'].forEach((mode) => {
+        beforeEach(() =>
+          cy.visit(
+            `/iframe.html?globals=theme:${theme}&id=lookupcomponent-lookup--lookup-${mode}`
+          )
+        );
+        it(`should render the ${mode} lookup component`, () => {
```

Comment:
Only 4 of your screenshots are uploading - we need to figure that out. 

One thing - I had some issues when I did a `beforeEach` in a `forEach` like this without an inner describe because it creates multiple `beforeEach` calls (one for each loop) and then each of them are executed before each test - causing the browser to be funny.

But I think there is more going on - as only 4 screenshots were uploaded.

Reply:
omg i know why this is 😂 I have a stray `.only` that i missed removing